src/client/index.js

```diff
@@ -132,6 +132,11 @@ class Hammerhead {
 
 var hammerhead = new Hammerhead();
 
-window.Hammerhead = hammerhead;
+Object.defineProperty(window, '@hammerhead@', {
```

Comment:
We are using `@hammerhead@` in the src attribute values. '%' is a URI reserved char:
https://tools.ietf.org/html/rfc3986#section-2.1
This is potentially dangerous, for example, our qunit tests fall while trying to decode url. I suggest using '@' char.